roles/realm_join/tasks/realm_join_ad.yml

```diff
@@ -0,0 +1,121 @@
+---
+- name: Ensure pexpect to answer password
+  become: yes
+  package:
+    name: pexpect
+    state: latest
+  # on Fedora this will be True due to the major version. Not nice, but works.
+  when: ansible_os_family != "RedHat" or ansible_distribution_major_version is version('8', '>')
```

Comment:
this evaluates to: "on debian" or "on all fedoras" or "on EL8+"
is that what you wanted?

Reply:
probably not it I see now that it differs on debian and RHEL.